elements/howto-tabs/howto-tabs.js

```diff
@@ -92,6 +108,15 @@
     }
 
     /**
+     * `disconnectedCallback` removes the event listeners that
+     * `connectedCallback` added.
+     */
+    disconnectedCallback() {
+      this.removeEventListener('keydown', this._onKeyDown);
+      this.removeEventListener('click', this._onClick);
```

Comment:
Good question. `click` triggers for touch events. Unless we want to handle touch interactions separately, I’d say we are good. But this is up for discussion.